CommunicationLayer/MessagingFramework/MessageDecodingHelpers.h

```diff
@@ -27,7 +27,14 @@
 
 #include <QByteArray>
 
-namespace MessageDecodingHelpers
+class MessageDecodingHelpers
 {
-   float getFloat(const QByteArray& data, int startIndex);
+public:
+    float getFloat(const QByteArray& data, int startIndex);
+    unsigned short getUnsignedShort(const QByteArray& data, int startIndex);
+    enum Type { FLOAT = 1, UNSIGNED_SHORT = 2 };
```

Comment:
I realize now the enum is only being used in private scope, I will be sure to stop exposing it as public next chance I get